components/prism-arturo.js

```diff
@@ -0,0 +1,49 @@
+Prism.languages.arturo = {
+    'comment': /;.*/,
+
+    'character': {
+        pattern: /`.`/,
+        alias: 'char'
+    },
+
+    'string': {
+        pattern: /"(?:[^"\\]|\\.)*"/,
+        greedy: true
+    },
+
+    'label': {
+        pattern: /[\w]+\b\??:/i,
+        alias: 'class-name'
+    },
+
+    'literal': {
+        pattern: /'(?:[\w]+\b\??:?)/,
+        alias: 'boolean'
+    },
+
+    'type': {
+        pattern: /:(?:[\w]+\b\??:?)/,
+        alias: 'boolean'
+    },
+
+    'keyword1': {
+        pattern: /\b(?:all|and|any|ascii|attr|attribute|attributeLabel|binary|block|char|contains|database|date|dictionary|empty|equal|even|every|exists|false|floating|function|greater|greaterOrEqual|if|in|inline|integer|is|key|label|leap|less|lessOrEqual|literal|logical|lower|nand|negative|nor|not|notEqual|null|numeric|odd|or|path|pathLabel|positive|prefix|prime|regex|same|set|some|sorted|standalone|string|subset|suffix|superset|symbol|symbolLiteral|true|try|type|unless|upper|when|whitespace|word|xnor|xor|zero)\?/,
```

Comment:
Could we please have a dedicated `boolean` token for `false|maybe|true`?

Reply:
Done 😉